instance.tf

```diff
@@ -0,0 +1,17 @@
+data "aws_ami" "amzn-ami" {
+	most_recent = true
+	filter{
+	  name = "name"
+	  values = ["amzn2-ami-hvm-*-gp2"]
+	}
+	
+	owners = ["137112412989"] #AWS
+
+}
+
```

Comment:
Add a security group and assign it to the instance. Allow 0.0.0.0/0 (everyone) to port 22 (ssh)
https://www.terraform.io/docs/providers/aws/r/security_group.html